modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -113,7 +117,7 @@ BGVtbYk3ZHAtqXs=
 ----
 <1> For `<user_name>` and `<password>`, specify the user name and password that you configured for your registry.
 
-ifndef::openshift-origin[]
+ifndef::openshift-origin,oc-mirror-v2[]
```

Comment:
Same with all these. These are usually excluding openshift-origin because it shouldn't be on OKD. It doesn't seem appropriate to be excluding each of these steps for oc-mirror-v2 too. remember that 'ifndef' means to only show the following content if oc-mirror-v2 is NOT defined.

Again, please check the preview. The rendering is not correct because of the conditionals:

![image](https://github.com/openshift/openshift-docs/assets/7443887/0ec748f0-5952-46e4-9155-ef3168819dd3)